### PR TITLE
Address `semver` CVE-2022-25883 and other moderate severity vulnerabilities

### PR DESCRIPTION
**Pull request by konstantin-msft**
- Address `semver` CVE-2022-25883 and other moderate severity vulnerabilities.
- Pin lerna to v6 for msal-node-e2e.yml.

### CODE REVIEW

File: change/@azure-msal-angular-e7f4ed65-7fb7-4311-8e1b-9c6a3d8e3994.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (tnorling) [inline change/@azure-msal-angular-e7f4ed65-7fb7-4311-8e1b-9c6a3d8e3994.json]: Package-lock updates are `none` changes
```suggestion
  "type": "none",
```

File: change/@azure-msal-browser-4d6383f0-97e5-4c0e-8bae-a122367a921d.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (tnorling) [inline change/@azure-msal-browser-4d6383f0-97e5-4c0e-8bae-a122367a921d.json]:
```suggestion
  "type": "none",
```

File: change/@azure-msal-common-55db98e4-7286-46a5-ad42-db9895f3bf3e.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (tnorling) [inline change/@azure-msal-common-55db98e4-7286-46a5-ad42-db9895f3bf3e.json]:
```suggestion
  "type": "none",
```

File: change/@azure-msal-node-4405fcf4-21ec-4ffa-99d6-a100c1f6563a.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (tnorling) [inline change/@azure-msal-node-4405fcf4-21ec-4ffa-99d6-a100c1f6563a.json]:
```suggestion
  "type": "none",
```

File: change/@azure-msal-react-83dd70ee-5e13-476b-a7a4-1456665409e2.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (tnorling) [inline change/@azure-msal-react-83dd70ee-5e13-476b-a7a4-1456665409e2.json]:
```suggestion
  "type": "none",
```

**Review by hectormmg — APPROVED** [review]
LGTM, thanks for taking care of the security upgrades!

File: change/@azure-msal-angular-e7f4ed65-7fb7-4311-8e1b-9c6a3d8e3994.json (hunk 1 above)
Comment (tnorling) [reply change/@azure-msal-angular-e7f4ed65-7fb7-4311-8e1b-9c6a3d8e3994.json]: Looks like these all got changed back?

Comment (konstantin-msft) [reply change/@azure-msal-angular-e7f4ed65-7fb7-4311-8e1b-9c6a3d8e3994.json]: My bad, fixed.

**Review by tnorling — APPROVED** [review]

**Review by peterzenz — APPROVED** [review]

**Pull request merged**